src/org/jitsi/impl/neomedia/rtp/StreamRTPManager.java

```diff
@@ -168,14 +171,17 @@ public void putResumableStreamRewriter(
     public ResumableStreamRewriter getResumableStreamRewriter(Long ssrc,
                                                               boolean create)
     {
-        ResumableStreamRewriter rewriter = ssrcToRewriter.get(ssrc);
-        if (rewriter == null && create)
+        synchronized (ssrcToRewriter)
```

Comment:
I would rather synchronize the block than depend on the fact that currently it is only used by one thread (the method is public and used in jitsi-videobridge).